config.json

```diff
@@ -22,6 +22,28 @@
           "format": "twShadows"
         }
       ]
+    },
+    "mui": {
+      "transformGroup": "js/custom",
+      "buildPath": "./build/mui/",
+      "files": [
+        {
+          "destination": "typography.json",
+          "format": "muiTypography"
+        },
+        {
+          "destination": "misc.json",
+          "format": "twMisc"
+        },
+        {
+          "destination": "colors.json",
+          "format": "twColors"
+        },
+        {
+          "destination": "shadows.json",
+          "format": "twShadows"
```

Comment:
Should these formats all be prefixed with mui? If not, it would probably make sense to change them to something mutual like `colors`, `misc`, etc.

Reply:
It's specifically pulling in the Tailwind format I built out previously. I was toying around with it to see if they'd be recyclable, and it ended up working. So, since they're no longer TW specific, I could definitely update them to a more general name.